genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_reader :name, :items, :id
+
+  def initialize(name, id = rand(1..10_000))
+    @id = id
+    @name = name
+    @items = []
+  end
+
+  # item: Item class object
+  def add_item(item)
+    @items.push(item)
+  end
```

Comment:
- [ ] It will be great to ensure that you complete the association. It seems you forgot to add ` item.genre = self`. If you check the other classes example author and label, you have completed the association but it seems you forgot to add it in the `genre`

Reply:
there is no need for this.

Reply:
It works correct

Reply:
The relation to the genre is set in the item itself